GameMods.hpp

```diff
@@ -36,19 +30,17 @@
 #define ENH_USE_JAVA_LIGHT_RAMP // Use Java Beta 1.3 light ramp instead of flawed PE one
 #define ENH_RUN_DAY_NIGHT_CYCLE // Allow the day/night cycle to run.
 #define ENH_USE_OWN_AO          // Use own ambient occlusion engine - looks pretty much the same except it fixes the corners
-#define ENH_ADD_OPTIONS_PAUSE   // Add an 'options' button in the pause menu
-#define ENH_EXTRA_ITEMS_IN_INV  // Add extra items in a new 5th row in the inventory.
+#define ENH_ADD_OPTIONS_PAUSE   // Add an 'options' button in the pause menu=
```

Comment:
Going to commit to remove this stray equal sign

Reply:
I resolved this in the latest commit.